src/utils/handleVote.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios';
+
+function handleVote(e, setVoted, idToVote, voter, setVotes) {
+    console.log(e.target)
+    console.log(idToVote)
+    if(e.target.innerText === "^") {
```

Comment:
a better (more semantic, easier for others to understand) way to identify which button it is is to use IDs. I would change this as innerText of '^' means nothing to most of us